src/styles/blocks/style.scss

```
@@ -0,0 +1,82 @@
+.wrapper {
+  display: flex;
+  background-image: linear-gradient(
+    to bottom, 
+    #515908, 
+    #6c790c,
+    #96ac0e,
+    #aecb0f,
+    #c6eb12
+  );
+  height: 100vh;
+
+  &__clock {
+    margin: auto;
+  }
+}
+
+.clock {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: $size;
+  height: $size;
+  border: 3px solid $main-color;
+  border-radius: 50%;
+
+  &__container {
+    position: relative;
+    width: $container-size;
+    height: $container-size;
+    border: 3px dashed $main-color;
+    border-radius: 50%;
+
+  }
+
+  &__axis {
+    width: 20px;
+    height: 20px;
+    background-image: linear-gradient(to bottom, #17180e, #2a2848);
+    border-radius: 50%;
+    position: absolute;
+    z-index: 100;
```

```suggestion
    z-index: 100;
```
why `z-index` is 100?